src/policy.c

```diff
@@ -1110,6 +1152,62 @@ void qd_policy_amqp_open(qd_connection_t *qd_conn) {
 }
 
 
+void qd_policy_amqp_open_connector(qd_connection_t *qd_conn) {
```

Comment:
Should this function be called something like qd_policy_populate_conn_policy_settings or something like that. The connector has already been "opened"

Reply:
The connector connection hasn't really opened yet. In this function policy is approving or denying the connection. Sure, the second Open has arrived but policy may slam the connection closed for some reason. The name is similar to the peer function that handles the remote open for incoming connections.

Reply:
I'm leaving the name as-is. It looks good at the call site in container.c.

There are two hard problems in software engineering: cache invalidation, naming things, and off-by-one errors.